eng/terraform/README.md

```diff
@@ -0,0 +1,78 @@
+# Ed-Fi Performance Testing in Azure
+
+Terraform scripts to deploy performance testing infrastructure into Azure.
+
+# Description
+
+This terraform script is a recreation of the Ed-Fi performance testing Azure environment. This script assumes that you will at start with a resource group for all resources, and VPC(s)/subnet(s) for the VMs and other resources to reside in and connect to.
+
+The resources created are:
+
+- analytics perf testing VM
```

Comment:
Unexpected surprise to get all three testing environments 😁 .

I have zero experience with TerraForm. Is it easy for us to manage these three lab environments independently? Each is used by a different development team. We don't want one team trying to rebuild their environment, and accidentally destroying someone else's in the process.

Reply:
I was honestly just remaking whatever existed in the azure resource group and didnt realize these were separate resources, but this helps me understand. I can make some changes with this in mind (remove the analytics perf testing).